File: 01_Language/01_Functions/php/array_pop.php
<?php

$stack = array("orange", "banana", "apple", "raspberry");
$fruit = array_pop($stack);
print_r($stack);

$map = array("k1" => "orange", "k2" => "banana", "k3" => "apple", "k4" => "raspberry");
$fruit = array_pop($map);
print_r($map);
print_r($fruit);

File: 01_Language/01_Functions/python/array_pop.py
# coding: utf-8

def array_pop(arr):
    try:
        return arr.pop()
    except IndexError:
        return None


if __name__ == '__main__':
    stack = ["orange", "banana", "apple", "raspberry"]
    print(array_pop(stack))
    print(array_pop(stack))
    print(array_pop(stack))
    print(array_pop(stack))
    print(array_pop(stack))
